.circleci/config.yml

```diff
@@ -72,55 +72,55 @@ commands:
           when: always
 
 jobs:
-  run-specs-with-postgres:
+  run-specs:
     executor:
-        name: solidusio_extensions/postgres
-        ruby_version: '2.7'
+        name: solidusio_extensions/<<parameters.database>>
+        ruby_version: <<parameters.ruby_version>>
     steps:
       - setup
       - test-branch:
-          branch: v3.1
-          rails_version: "~> 6.1"
-      - solidusio_extensions/store-test-results
-  run-specs-with-mysql:
-    executor:
-        name: solidusio_extensions/mysql
-        ruby_version: '3.0'
-    steps:
-      - setup
-      - test-branch:
-          branch: v3.1
-          rails_version: "~> 6.1"
-      - solidusio_extensions/store-test-results
-  run-specs-with-postgres-and-solidus-latest:
-    executor:
-        name: solidusio_extensions/postgres
-        ruby_version: '3.0'
-    steps:
-      - setup
-      - test-branch:
-          branch: master
-          rails_version: "~> 7.0"
-      - solidusio_extensions/store-test-results
-  run-specs-with-mysql-and-solidus-latest:
-    executor:
-        name: solidusio_extensions/mysql
-        ruby_version: '3.1'
-
-    steps:
-      - setup
-      - test-branch:
-          branch: master
-          rails_version: "~> 7.0"
+          branch: <<parameters.solidus_branch>>
+          rails_version: <<parameters.rails_version>>
       - solidusio_extensions/store-test-results
+    parameters:
+      solidus_branch:
+        type: string
+        default: 'v3.2'
+      rails_version:
+        type: string
+        default: '~> 7.0'
+      ruby_version:
+        type: string
+        default: '3.1'
+      database:
+        type: string
+        default: 'postgres'
 
 workflows:
   "Run specs on supported Solidus versions":
     jobs:
-      - run-specs-with-postgres
-      - run-specs-with-mysql
-      - run-specs-with-postgres-and-solidus-latest
-      - run-specs-with-mysql-and-solidus-latest
+      - run-specs
```

Comment:
Maybe we could add a name to also describe this job, so that in the PRs status is immediately clear what's failing?

Reply:
Hmm... I was initially hesitant that naming the base job as `run-specs-on-solidus-3-2-rails-7-ruby-3-1-postgres` would be unwieldy, but it turns out that's it's kinda okay. I'll rename it to that.